### PR TITLE
Fix Issue #35

### PR DESCRIPTION
**Pull request by Piran-K**
Add feature which allows user to set range of random delay.
Change config.json.example file. Set value of random_delay in json file to [0,0] which is equivalent to no delay.
Update instructions in README.

### CODE REVIEW

**Comment by TobiasPankner** [comment]
Looks pretty good. One issue is:  
User pulls newest version.  
Doesn't update their config file.  
Exception because the config value is bool and not an array.  

You could add a check if the value is a bool and then inform to update the delay value.  
Or you could also support the old functionality (bool) in parallel.

**Comment by Piran-K** [comment]
I have tested it out with random_delay in config.json set to: true, false, [0,0], [-5,-1], [20,20], [10,31].

**Comment by TobiasPankner** [comment]
Can you please take a look at the comment I added?

**Comment by Piran-K** [comment]
> Can you please take a look at the review comment I added?

Upon your suggestion, I have added support for the old functionality in parallel.

**Comment by TobiasPankner** [comment]
Sorry, my bad. Forgot to finsish the review.

File: auto_joiner.py [hunk 1]
```diff
@@ -443,10 +443,22 @@ def join_meeting(meeting):
...
+            print(f"Please update the random_delay in config.json file as per latest instructions in README")
+            if config['random_delay']:
+                delay = random.randrange(10, 31, 1)
+            else:
+                delay = 0
+        else:
+            if config['random_delay'][0] == config['random_delay'][1]:
+                delay = config['random_delay'][0]
+            else:
+                delay = random.randrange(config['random_delay'][0], config['random_delay'][1], 1)
```
[the remark this answers is not in the document]
Comment (Piran-K) [reply auto_joiner.py]: I have changed it now.

**Comment by TobiasPankner** [comment]
Great, thanks for the contribution!

**Pull request merged**